web_hide_db_manager_link/README.rst

```diff
@@ -1,37 +1,57 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+=============================================
 Hide link to database manager in login screen
 =============================================
 
 This module hides the "Manage Databases" link at the bottom of login screen.
 
-The image in *web_hide_db_manager_link/static/src/img/screen.png* shows the resulting loging screen.
-
-.. image:: web_hide_db_manager_link/static/src/img/screenshot.png
 
 Installation
 ============
 
 To install this module, you need to:
 
  * Go to Settings / Local Modules
- * Search by module name "Hide link to database manager in login screen" or by module technical name "*web_hide_db_manager_link*" 
+ * Search by module name "Hide link to database manager in login screen" or
+   by module technical name "*web_hide_db_manager_link*"
  * Click install button
 
+Usage
+=====
+
+When this module is installed the link "Manage Databases" will be hidden in
+the login screen.
+
+The image in *web_hide_db_manager_link/static/src/img/screen.png* shows the
+resulting loging screen.
+
+.. image:: web_hide_db_manager_link/static/src/img/screenshot.png
+
+In order to access to the database management, the administrator should use a
+specific URL <myserver>/web/database/manager
+
+.. image:: https://odoo-community.org/website/image/ir
+.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/9.0
+
 Known issues
 ============
 
-As Odoo v8 templates only live in the database, this module must be installed in each database you want this to be in effect.
+* As Odoo v8 templates only live in the database, this module must be
```

Comment:
@sbidoul Yes, still the case in v9.

The login screen is generated from a template, and need to fetch data from some database. It will therefore be created using the definition set in the catched DB. When you switch to another DB, if that module is not installed the link will be visible again.

The database manager (web/database/manager) is, in contrast, a static HTML page and, as such, does not depend on the DB to render it's content.